client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -189,21 +193,43 @@ const InstalledInOrPricing = ( {
 	currentSites,
 } ) => {
 	const translate = useTranslate();
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const installedPlugins =
+		useSelector( ( state ) => getInstalledPlugins( state, [ selectedSiteId ] ) ) || [];
```

Comment:
`getInstalledPlugins` didn't work properly due to some plugins mismatch in slugs. I have used `getPluginOnSites` which contains `isEqualSlugOrId` and properly matches the current plugin with the installed.

https://github.com/Automattic/wp-calypso/blob/bbe31df69fea04abd0b7d3df71aabb4fa597d693/client/state/plugins/installed/selectors.js#L108-L110

https://github.com/Automattic/wp-calypso/pull/61485

Reply:
> plugins mismatch in slugs

Why are slugs not matching?

Reply:
```
`woocommerce-com-` prefix is added to track plugin update checks pinged to WooCommerce.com, and to not interfere with WordPress Core’s plugin update check.
```

You can read more here p9o2xV-1Of-p2#comment-5485

Reply:
Well... that raises a lot more questions than it answers :laughing: Thanks for the link!